main.js

```diff
@@ -49,4 +49,8 @@ var viewer;
 
     createBrowserApi(opts).then(initViewer);
   });
+
+  global.pdfLoaded = new Promise(function(resolve, reject) {
```

Comment:
This seems redundant, we already get notified when the viewer is ready with `setLoadCallback`. https://github.com/electron/electron/blob/master/spec/chromium-spec.js#L1004